tests/lib/nested.sh

```diff
@@ -45,9 +45,9 @@ nested_get_boot_id() {
 
 nested_wait_for_reboot() {
     local initial_boot_id="$1"
-    local retry wait last_boot_id
-    retry=150
-    wait=5
+    local last_boot_id="$initial_boot_id"
```

Comment:
This is set to "" below in line 50, so it's enough to just declare it here and remove line 50?

Reply:
ah I missed that sorry you're right that line should be removed

Reply:
fixed